src/features/Apiexplorer/LoginDialog/index.tsx

```diff
@@ -34,7 +34,14 @@ export const LoginDialog = ({ setToggleModal }: TLoginDialog) => {
             has_close_button
             className={styles.wrapper}
           >
-            <div className={styles.modal}>Log in or sign up to continue.</div>
+            <div className={styles.modal}>
+              This API call must be authorised because it requires access to your account
+              information.
+            </div>
+            <div className={styles.modal}>
+              Log in to your Deriv account to proceed. If you don’t have a Deriv account, sign up
+              first.
+            </div>
```

Comment:
Perhaps, it is possible to put these 2 sentences in two seperate `<p>` elements? I believe that makes more sense and I think we can then remove the padding from the `.modal` class.

Reply:
I'll change `<div>` to `<p>` but we will still need the padding.

Reply:
Sure, that is no problem